examples/12.cancel_all_open_orders.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Cancels all open orders for the given market
+ */
+
+/* eslint-disable no-console */
+import {
+  ORDER_STATUS,
+  ORDER_SIDE,
+  // MinifiedCandleStick,
+  ORDER_TYPE,
+  toBaseNumber,
+  MinifiedCandleStick,
+  Faucet,
+  OrderSigner,
+  parseSigPK,
+  ADJUST_MARGIN,
+} from "@firefly-exchange/library-sui";
+import { Networks, BluefinClient } from "../index";
+
+async function main() {
+  const dummyAccountKey =
+    "trigger swim reunion gate hen black real deer light nature trial dust";
+
+  const client = new BluefinClient(
+    true,
+    Networks.TESTNET_SUI,
+    dummyAccountKey,
+    "ED25519"
+  ); //passing isTermAccepted = true for compliance and authorizarion
+  await client.init();
+  let symbol = "ETH-PERP";
```

Comment:
For the purpose of example use different markets for both orders. Place on order on ETH-PERP and other on BTC-PERP